packages/picker/src/Picker.ts

```diff
@@ -406,6 +411,17 @@ export class PickerBase extends SizedMixin(Focusable) {
         ];
     }
 
+    customSetAttribute = (attribute: string, value: string): void => {
+        if (attribute !== 'aria-label') {
+            HTMLElement.prototype.setAttribute.apply(this.button, [
+                attribute,
+                value,
+            ]);
+            return;
+        }
+        this.appliedLabel = value;
```

Comment:
Could `this.appliedLabel` be assigned alongside `this.value` whenever the selected value changes, instead of overriding `setAttribute`?

If not, like maybe a timing issue for VO to pick up the new value, a comment here explaining this approach would probably be useful down the line.

Reply:
@hunterloftis I swapped the approaches here to something that I think is less abusive.